victkarangwa/psql-backup-cron
# Add send db backup file to specific email

closes https://github.com/victkarangwa/psql-backup-cron/issues/1

**How to test this functionality locally?**
- Add the following env vars: 

```
       GMAIL_ACCOUNT_EMAIL={gmail_account_that_is_configured_to_use_nodemailer}
       GMAIL_ACCOUNT_PASSWORD={password_of_the_gmail_account}
       BACKUP_EMAIL={email_to_receive_a_backup_as_attachment}
```
- Modify the crontab to run from every Friday to every minute

- Once that is done after 1 minute, the email should be on its to the backup email provided. Screenshot attached below for my test:

![backup file](https://user-images.githubusercontent.com/51243858/132505282-3971d726-9fea-45dd-8067-0e8faa872f60.png)
